src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -9093,6 +9093,7 @@ public static partial class RuntimeFeature
     }
     public static partial class RuntimeHelpers
     {
+        [System.ObsoleteAttribute("OffsetToStringData is obsolete. Use string.GetPinnableReference() instead.")]
```

Comment:
Should we be following @terrajobst's new plan for obsoletion?

Reply:
It's a good question. :)